### PR TITLE
fix the DB connection

### PR DESCRIPTION
**Pull request by MxmIv**
Fixing Database connection. 

Make sure your table is called "note" with lower case.

### CODE REVIEW

File: HaulageApplication/HaulageApp/Views/NotePage.xaml.cs [hunk 1]
```diff
@@ -6,7 +6,7 @@ public partial class NotePage : ContentPage
 {
     public NotePage(NoteViewModel viewModel)
     {
-        this.BindingContext = viewModel;   
         InitializeComponent();
-    }
-}
\ No newline at end of file
+        BindingContext = viewModel;
```
Comment (tiphainetiphaine) [inline HaulageApplication/HaulageApp/Views/NotePage.xaml.cs:10]: why was this necessary?

File: HaulageApplication/HaulageApp/Data/HaulageDbContext.cs [hunk 2]
```diff
@@ -10,6 +11,6 @@ public HaulageDbContext()
         public HaulageDbContext(DbContextOptions options) : base(options)
         { }
 
-        public DbSet<Note> Notes { get; set; }
+        public DbSet<Note> note { get; set; }
```
Comment (MxmIv) [inline HaulageApplication/HaulageApp/Data/HaulageDbContext.cs:14]: this is the change which fixes the connecting

Comment (tiphainetiphaine) [inline HaulageApplication/HaulageApp/Data/HaulageDbContext.cs:14]: so the name of the database?

**Review by FMulligan — APPROVED** [review]

File: HaulageApplication/HaulageApp/Data/HaulageDbContext.cs (hunk 2 above)
Comment (MxmIv) [reply HaulageApplication/HaulageApp/Data/HaulageDbContext.cs:14]: yes

**Review by tiphainetiphaine — APPROVED** [review]
I'm unsure what the difference really is other than the additional logs (added some comments) but I'll approve, then we can review and tidy :) thanks!
